exercises/allergies/AllergiesTest.fs

```diff
@@ -1,74 +1,79 @@
-﻿module AllergiesTest
+module AllergiesTest
 
 open System
 open NUnit.Framework
+open FsUnit
 open Allergies
 
 [<Test>]
 let ``No allergies means not allergic`` () =
-    Assert.That(allergicTo Allergen.Peanuts 0, Is.False)
-    Assert.That(allergicTo Allergen.Cats 0, Is.False)
-    Assert.That(allergicTo Allergen.Strawberries 0, Is.False)
+    allergicTo Allergen.Peanuts 0 |> should equal false
+    allergicTo Allergen.Cats 0 |> should equal false
+    allergicTo Allergen.Strawberries 0 |> should equal false
 
 [<Test>]
 [<Ignore("Remove to run test")>]
 let ``Allergic to eggs`` () =
-    Assert.That(allergicTo Allergen.Eggs 1, Is.True)
+    allergicTo Allergen.Eggs 1 |> should equal true
```

Comment:
@robkeim What do you think about using `should equal true`? The docs also mention `should be True`, but I was hesistant to use that option as the uppercase T in `True` could throw newcomers off I think,

Reply:
Given there's already a mix of `should be` and `should equal` I'm fine going with `should equal true`.